CONTRIBUTING.md

```diff
@@ -159,7 +159,7 @@ NOTE: Partially copied from https://raw.githubusercontent.com/emberjs/ember.js/m
 
 Have you got enough knowledge in a specific feature and want to help with docs?
 Ember-cli documentation lives at the branch
```

Comment:
Exactly.

But the version within this repository isn't live yet. I thought already updating those URLs might be great because all changes will be made in the new repo (So that we don't have to merge the "gh-pages" branch into the new repo later. I think this could cause a lot of trouble since I will change it's structure completely.)